subjects/devops/input-redirection/README.md

```diff
@@ -0,0 +1,43 @@
+## Input redirection
+
+### Instructions
+
+In this exercise you will make a script `input-redirection.sh`.
+This script will read from an here document (`HereDoc`).
```

Comment:
to keep the naming convention it might be better to name `HereDoc` `here-doc`

Reply:
Here I think we should use the common standard instead of our own convention. `HereDoc` was found in many articles and it is my preferred way, to be more neutral I'll pick one of the shown abbreviations in wikipedia, which will be `heredoc` :).